pages/docs/reason-compiler/latest/decorators.mdx

```diff
@@ -808,7 +848,49 @@ log(404, "NOT_FOUND", "this", "page", "is", "not", "found");
 ```
 
 ## `[@bs.inline]`
-TODO
+
+Especially for library authors, `bs.inline` could be interesting. It tells the compiler explicitly to inline a constant directly. 
```

Comment:
Maybe explaining the problem first? Something like:
```
In Reason/OCaml you can't export let variables (this is the opposite as we are used to in JS). Often you can export modules, but If you want to export a single value, you can use [bs.inline].
```

Reply:
That's not the case? let vars are exported the same way as let functions? It's only a helper to get zero-cost bindings like [here](https://github.com/BuckleScript/bucklescript/issues/3472#issuecomment-480326301).

Reply:
Let me rephase it: 

In Reason/OCaml you can export variables (int/string/primitives types) evaluated (or "replaced with the value") instead of the reference to the variable with [bs.inline]. Often as a library author you can encounter the need of it. 

For example: <<<Example here>>>>

I mean, I prefer to read the problem first, is easier to index (since its logically to look for) and makes the lecture of the entire section like an explanation rather than docs as a list of typedefs.

I might mess around with the reasoning but I got confused on what inline was doing.